cairo-contracts/packages/contracts/src/erc20.cairo

```diff
@@ -32,6 +34,9 @@ pub mod ERC20Mintable {
         mintable: ERC20MintableComponent::Storage,
         #[substorage(v0)]
         erc20: ERC20Component::Storage,
+        // The decimals value is stored locally in the contract.
```

Comment:
Came across this update: https://github.com/OpenZeppelin/cairo-contracts/pull/1294
Hopefully, it'll be available soon in their next release, so we can go with that and won't need to keep this internal state anymore.

Reply:
Yes I saw this too 👍🏼 but this is more in the `static approach`. The `storage approach` still remains the same.

Reply:
I didn't open an issue for this, as the storage approach stayed the same.